pkg/gather/ssh/ssh.go

```diff
@@ -127,6 +127,7 @@ func LoadPrivateSSHKeys(paths []string) ([]interface{}, error) {
 			errs = append(errs, errors.Wrapf(err, "failed to parse SSH private key from %q", path))
 			continue
 		}
+		logrus.Debugf("Read private SSH key from %s", path)
```

Comment:
at this point we are using the SSH key from the path, no read is not accurate.

Reply:
> at this point we are using the SSH key from the path, no read is not accurate.

This is just after we read the SSH key from disk, so I think the wording is accurate.  Although there is the problem that "read" could be... read... as present tense (sounds like reed), when I'm meaning past tense (sounds like red).  Did you have an alternative wording suggestion?

Reply:
> This is just after we read the SSH key from disk

there is read file here https://github.com/openshift/installer/pull/2403/files#diff-026402989453f294b4fa16c2936e1209R120

there is load private key here https://github.com/openshift/installer/pull/2403/files#diff-026402989453f294b4fa16c2936e1209R125

and the the debug statement... so saying `Read (red) private SSH key` here is not great, but `Using/Loaded private key` more be more appropriate imo.

Reply:
Also with https://github.com/openshift/installer/pull/2403/files#diff-bf0061f9e327ec59aa150a74e75c7b91R46 we should maybe just drop this??

Reply:
Dropped in 818501f0e -> 90064da41.